lib/bambora/profile.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Bambora
+  class Profile < JSONRequest
```

Comment:
More of a semantic thing, I find it a bit confusing that `Profile` inherits from `JSONRequest`. I infer from this inheritance chain that `Profile` is a type of `JSONRequest`, when in reality it contains multiple methods that can make JSON requests. This seems to me to be more like a concern/module.

From the perspective of the caller though, when you are using this client, you are only ever using one API method at a time, so from the point of view of the caller, it does only represent one JSON request.

Now I've just argued with myself, haha.

Reply:
I was going to call this `Domain` since it represents the Bambora domain, but then I realized that the domain also uses XML. I could perhaps name this `JSONDomain` and `XMLDomain`.